authorship/data.py

```diff
@@ -60,25 +60,24 @@ def store_data(path, speakers):
         f.write(json.dumps(speakers))
 
 
-all_words = 0
-
 def read_session(lines):
-    global all_words
-    global test
     speakers = []
     test = lines.read()
```

Comment:
test?

Reply:
not sure why it was global or given this name, i changed it to lines_to_words